conf/steps/panel_prep.config

```diff
@@ -25,27 +25,41 @@ process {
             "--no-version"
         ].join(' ')
         ext.prefix = { "${meta.id}_chrrename" }
+        publishDir = [ enabled: false ]
     }
 
-    withName: 'NFCORE_PHASEIMPUTE:PHASEIMPUTE:GET_PANEL:.*' {
-        publishDir = [
-            path: { "${params.outdir}/prep_panel/" },
-            mode: params.publish_dir_mode,
-            enabled: true,
-            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
-        ]
+        withName: 'NFCORE_PHASEIMPUTE:PHASEIMPUTE:VCF_NORMALIZE_BCFTOOLS:BCFTOOLS_NORM' {
+        ext.args      = '-m +any --no-version --output-type z'
+        ext.prefix    = { "${meta.id}_${meta.chr}_multiallelic" }
+        cpus          = 2
```

Comment:
Why do you add the cpu, memory and maxRetries each time ?
I'm not sure this is necessary 🤔